test/e2e/test-specs.yml

```diff
@@ -17,9 +17,26 @@ scenarios:
     before:
       - cd ../../ && NR_PROM_LICENSE_KEY=${LICENSE_KEY} NR_PROM_CLUSTER=${SCENARIO_TAG} make tilt-ci HELM_VALUES_FILE="./test/e2e/values-multi-shards.yaml"
     after:
-      - kubectl logs -l app.kubernetes.io/name=newrelic-prometheus --all-containers --prefix=true
+      - kubectl logs -l app.kubernetes.io/name=newrelic-prometheus-agent --all-containers --prefix=true
       - kubectl get all -o wide
       - cd ../../ && tilt down
     tests:
       metrics:
         - source: "metrics.yml"
+  - description: Scrape test environment metrics from curated experience only
```

Comment:
If time is a concern since this use case is not important enough we could:
 - merge sharding and curated experience e2e tests to reduce the time
 - just remove it